scripts/validate.sh

```diff
@@ -47,27 +47,27 @@ fi
 
 echo "Verifying that the package versionlocks are correct..."
 
-function versionlock-entries() {
+versionlock-entries() {
```

Comment:
We've taken the approach of including *all* the optional shell grammar for function definitions (including the `function` keyword, parentheses after the `fname` and curly braces around the compound command). Please don't remove this.

Reply:
sure, I simply trying to make all the function-style in the same way under `./scripts/` folder.